2016-host-organization-application/index.css

```diff
@@ -46,6 +46,10 @@
   padding: 0px 50px 25px 85px;
 }
 
+.sub-goal-points {
+  margin-left: 20px;
```

Comment:
Bullets should have a natural padding/margin thingy already. Something might be removing it, then we add it back. Seems a bit roundabout. Would the natural spacing be enough/too much? I don't think it's 20px, probably much more.